snpseq_metadata/models/converter.py

```diff
@@ -41,6 +41,15 @@
 
 
 class Converter:
+    """
+    The main class for doing conversions between models. The idea is that the conversion is made by
+    calling the appropriate class method and supplying the model instance to convert from. This
+    pattern allows the models to be implemented independently of each other, i.e. the model modules
+    themselves need not know anything about any other model.
+
+    Example:
+        sra_library_instance = Converter.ngi_to_sra(ngi_library_instance)
```

Comment:
I'm not sure what's the meaning of ngi-models. Is it the models used in the genologics repository?

Reply:
well, the naming of things may not be the best.. I started out with the NGI models meant to represent the metadata as we, NGI, see it. The sources for this data are, as you know, a mix between clarity and Illumina run folder output. On top of that came the LIMSmodel which is more directly tied to clarity and what your service exports.